### PR TITLE
[PLAT-965] Adds groups documentation

### PR DESCRIPTION
**Pull request by lily-tsou**
- Adds documentation and examples for all groups functions to README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -638,7 +638,137 @@ async function main() {
 
 main()
 ```
+### Groups
```
Comment (rosasck) [inline README.md:641]: TozStore Groups

File: README.md [hunk 2]
```diff
@@ -638,7 +638,137 @@ async function main() {
...
+Groups allow users to share encrypted data with multiple clients, without having to manually share each record.<br>
+Any client can create a group and can be added to an existing group. The creator of the group will automatically be given management capabilities, and may optionally pass in read and/or write capabilities during group creation.<br><br>
+Create a group:
```
Comment (rosasck) [inline README.md:644]: make this a heading

File: README.md [hunk 3]
```diff
@@ -638,7 +638,137 @@ async function main() {
...
+  memberCount
+},
+capabilities: {}
+```
+These fields can be used as parameters to various groups functions. The group ID is particularly useful for this.<br>
+Get the groupID from the GroupMembership object returned:
+```js
+const groupID = example_group.group.groupID
+```
+Add one or more group members:
```
Comment (rosasck) [inline README.md:687]: make this a heading

**Review by galxy25 — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -638,7 +638,137 @@ async function main() {
...
+    members = await client.listGroupMembers(groupID)
+    console.log("After deleting group member, members = \n", members)
+  }
+}
+catch (e){
+  console.error(e)
+}
+```
+```removeGroupMembers()``` will return true if removal was successful and will otherwise throw an error.<br><br>
+Get a single group's info:
```
Comment (rosasck) [inline README.md:722]: do the same for all

**Review by rosasck — APPROVED** [review]

**Review by Mrhea — APPROVED** [review]
Pending changes

**Pull request merged**